src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -727,6 +727,37 @@ impl<T: AstInfo> AstDisplay for AwsConnectionOption<T> {
 }
 impl_display_t!(AwsConnectionOption);
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum AwsPrivateLinkConnectionOptionName {
+    ServiceName,
+    AvailabilityZones,
+}
+
+impl AstDisplay for AwsPrivateLinkConnectionOptionName {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        f.write_str(match self {
+            AwsPrivateLinkConnectionOptionName::ServiceName => "SERVICE NAME",
+            AwsPrivateLinkConnectionOptionName::AvailabilityZones => "AVAILABILITY ZONES",
+        })
+    }
+}
+impl_display!(AwsPrivateLinkConnectionOptionName);
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+/// An option in a `CREATE CONNECTION...AWS PRIVATELINK`.
+pub struct AwsPrivateLinkConnectionOption<T: AstInfo> {
+    pub name: AwsPrivateLinkConnectionOptionName,
+    pub value: WithOptionValue<T>,
+}
+
+impl<T: AstInfo> AstDisplay for AwsPrivateLinkConnectionOption<T> {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        f.write_node(&self.name);
+        f.write_node(&self.value);
```

Comment:
Do we want some form of separator in-between these? Like `<name>: <value>`?

Reply:
Now that I look every other option has the following format: `<name> = <value>`.

Reply:
If I add the equals, then it fails to reparse in the cargo tests. I can add the equals sign if I make it `Option`al, but it seems weird to make something optional when it's required.

```
reparse failed: Expected option value, found equals sign
```

Reply:
@alex-hunt-materialize I was just thinking about this, shouldn't there at least be a space separating the name and value?